### PR TITLE
Fix disconnect when pid's do not match

### PR DESCRIPTION
**Pull request by paupujolasparset**

### CODE REVIEW

File: disconnect/action.yaml [hunk 1]
```diff
@@ -20,10 +20,12 @@ runs:
             if [ "$pid" == "${{ inputs.pid }}" ]; then
               echo "Killing process ${{ inputs.pid }}"
               sudo kill ${{ inputs.pid }}
+            else
+              echo "pid (${{ inputs.pid }}) provided in inputs does not match with the running OpenVPN process ($pid).
+              exit 1
             fi
           else
-            echo "Your pid $pid is not the OpenVPN pid running in the instance"
```
Comment (llorenc-aizon) [inline disconnect/action.yaml:25]: This message is not correct.... _Your pid_ should be input pid, but none has been provided

File: disconnect/action.yaml [hunk 2]
```diff
@@ -20,10 +20,12 @@ runs:
...
-            echo "Killing process OpenVPN pid $pid"
+            echo "No pid provided in inputs. Killing process OpenVPN pid $pid"
```
Comment (llorenc-aizon) [inline disconnect/action.yaml:28]: This message sounds redundant to me... either this or just the previous one

**Review by llorenc-aizon — CHANGES_REQUESTED** [review]

File: disconnect/action.yaml (hunk 1 above)
Comment (paupujolasparset) [reply disconnect/action.yaml:25]: That's why I deleted the message

File: disconnect/action.yaml (hunk 2 above)
Comment (paupujolasparset) [reply disconnect/action.yaml:28]: Which is the previous one?

**Review by llorenc-aizon — APPROVED** [review]

**Review by christiangda — APPROVED** [review]

**Pull request merged**